api/cpp/include/slint_brush.h

```diff
@@ -113,4 +122,41 @@ Color Brush::color() const
     return result;
 }
 
+inline Brush Brush::brighter(float factor) const
+{
+    Brush result = *this;
+    switch (data.tag) {
+    case Tag::SolidColor: {
+        cbindgen_private::types::slint_color_brighter(&data.solid_color._0, factor,
+                                                      &result.data.solid_color._0);
+        break;
+    }
+    case Tag::LinearGradient:
+        for (std::size_t i = 0; i < data.linear_gradient._0.size(); ++i) {
+            cbindgen_private::types::slint_color_brighter(&data.linear_gradient._0[i].color, factor,
+                                                          &result.data.linear_gradient._0[i].color);
+        }
+        break;
+    }
+    return result;
+}
+
+inline Brush Brush::darker(float factor) const
+{
+    Brush result = *this;
+    switch (data.tag) {
+    case Tag::SolidColor: {
+        cbindgen_private::types::slint_color_darker(&data.solid_color._0, factor,
+                                                    &result.data.solid_color._0);
+        break;
+    }
+    case Tag::LinearGradient:
+        for (std::size_t i = 0; i < data.linear_gradient._0.size(); ++i) {
```

Comment:
Skip the first entry of the gradient, which is the angle:

```suggestion
        for (std::size_t i = 1; i < data.linear_gradient._0.size(); ++i) {
```